src/UI/Canvas.cs

```diff
@@ -0,0 +1,162 @@
+using TileMapper.Windowing;
+using Raylib_cs;
+using ImGuiNET;
+using System.Numerics;
+
+namespace TileMapper {
+    
+    public class Canvas : GraphicsWindow {
+
+        int mapHeight, mapWidth;
```

Comment:
Private variables should begin with an underscore (and just be marked as private). See the C# guidelines.